nyc_taxi/anaconda-project.yml

```diff
@@ -15,21 +15,20 @@ examples_config:
 user_fields: [examples_config]
 
 channels:
-- pyviz/label/dev
 - defaults
 
 packages: &pkgs
 - python=3.11.8
-- notebook>=7.0.8
+- notebook<7
 - bokeh>=3.4.0
 - datashader
```

Comment:
Can you please add a min version to datashader?

Reply:
Done